modules/network/basic-cdk/integ/integ_network.py

```diff
@@ -22,7 +22,10 @@
             project_name="integ",
             deployment_name="testing",
             module_name="basic-cdk",
+            vpc_cidr="10.0.0.0/16",
+            cidr_mask=24,
             internet_accessible=True,
+            local_zones=[],
```

Comment:
Leaving as empty as region cannot be determined during synth. Earlier solution of using `AWS_REGION` env variable to load AZs using EC2 boto3 session worked for `us-east-1`, but would not work for other regions as it would still return `us-east-1` AZs.

I also attempted to use `cdk.Aws.REGION` and append the AZs (a, b, c etc) to the end of the region, so that it could use tokens to resolve during deployment. However the local zone is used in the ID of the subnet, which would lead to an error like this during synth `RuntimeError: Error: Resolution error: ID components may not include unresolved tokens: LocalZonePrivate-${Token[AWS.Region.12]}a.`